### PR TITLE
update setup script

### PR DESCRIPTION
**Pull request by sixshotx**
Update the setup scripts:
1, correct the aarch tool name
2, test apt-get for non-ubuntu system

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -46,10 +46,16 @@ echo "\n\nDown QEMU OS images? This will take a while. (Y/n):"
 read inp
 
 # Install dependencies
-echo "Installing dependencies..."
-echo "sudo apt-get -y build-dep qemu"
-sudo apt-get -y build-dep qemu
-#sudo apt-get -y install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+if hash apt-get 2>/dev/null; then
```
Comment (pranith) [inline setup.sh:49]: I think it is better to use "command -v" like you used in line 18 above.

**Comment by sixshotx** [comment]
I agree that using "command -v" could be more posix compatible. The "hash" way is also a common to check the command existence in bash according to [ref](http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script).

It seems that the build is failed due to a timeout?

**Comment by pranith** [comment]
OK. The tests passed after restarting it. Merged now. Thanks!

**Comment by sixshotx** [comment]
Can you merge the change to v2.3 for the lps demo?

**Comment by pranith** [comment]
I created a new point release v2.3.1 with the update merged.

https://github.com/gtcasl/qsim/releases/tag/v2.3.1

**Pull request merged**